TestProjects/BatchRendererGroup_HDRP/ProjectSettings/VFXManager.asset

```diff
@@ -10,6 +10,7 @@ VFXManager:
   m_RenderPipeSettingsPath: 
   m_FixedTimeStep: 0.016666668
   m_MaxDeltaTime: 0.05
-  m_CompiledVersion: 4
+  m_MaxScrubTime: 30
```

Comment:
Are these changes to the VFXManager.asset file intended?
I don't think it's a big deal if they're pushed though, I guess they're part of the regenerated files when opening the project in Unity.

Reply:
Same for all the .asset files in the other projects :)

Reply:
Yes, they are introduced by opening the project with this editor version to update the lockfiles. With regards to the `staging/` branches: since the editor automatically updates them when opening, should those be backed out?